### PR TITLE
chapter_9

### PR DESCRIPTION
**Pull request by daniadore**
Ready

### CODE REVIEW

**Comment by daniadore** [comment]
Ready

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -4,8 +4,9 @@ def new; end
   def create
     user = User.find_by email: params[:session][:email].downcase
```
Comment (huongnt-2545) [inline app/controllers/sessions_controller.rb:5]:
```suggestion
    @user = User.find_by email: params[:session][:email].downcase
    if @user&.authenticate params[:session][:password]
```

**Comment by daniadore** [comment]
Ready

File: app/models/user.rb [hunk 2]
```diff
@@ -1,11 +1,13 @@
 class User < ApplicationRecord
+    attr_accessor :remember_token
```
Comment (ngocvt-0484) [inline app/models/user.rb:2]: wrong indent

**Comment by daniadore** [comment]
Ready

**Review by thaodtd-0665 — APPROVED** [review]

**Comment by thaodtd-0665** [comment]
merged

**Pull request merged**